site/content/en/docs/Concepts/principles.en.md

```diff
@@ -5,14 +5,13 @@ date: 2019-06-18T15:31:58+08:00
 
 The primary goal of minikube is to make it simple to run Kubernetes locally, for day-to-day development workflows and learning purposes. Here are the guiding principles for minikube, in rough priority order:
 
-1. User-friendly and accessible
-2. Inclusive and community-driven
-3. Cross-platform
-4. Support all Kubernetes features
-5. High-fidelity
-6. Compatible with all supported Kubernetes releases
-7. Support for all Kubernetes-friendly container runtimes
-8. Stable and easy to debug
+1.  Inclusive and community-driven
+1.  User-friendly
+1.  Support all Kubernetes features
+1.  Cross-platform
+1.  Reliable
+1.  High Performance
```

Comment:
I am not sure if "high performance" should be on the principles at all. While I like the solution to be as effecient as possible, I am not sure it is more important than "developer focused" ?

One example: if we replaced `dockerd` with `containerd` it would improve runtime performance (microk8s did this). But it would also make it harder / more cumbersome to use when developing.

Reply:
I feel like "High Performance" should stay, as it provides guidance that we actually do care about performance. That said, we also care about "User-friendly" and "Reliable", so we need to balance principles against one another.